### PR TITLE
Update : Options --data [file] & --no-color + tips on --suppress-exit-code for CI/CD

### PR DESCRIPTION
**Pull request by Rom1deTroyes**
The `--data` option doesn't exist (newman --version : 4.5.7)

```bash
$ newman run mycollection.json --data data.json
error: unknown option '--data'
```

Works well with `--iteration-data` as specified in the help of newman :
```
$ newman run --help |grep "data file"
  -d, --iteration-data <path>     Specify a data file to use for iterations (either json or csv).
```

### CODE REVIEW

**Comment by SueSmith** [comment]
Hi @Rom1deTroyes! Thank you so much for this, I'm hearing that some of the other options have also changed - would you be happy to run `newman run -h` and update those too? 😬 🙌

**Comment by Rom1deTroyes** [comment]
Found `--color` who no longer exist.

Off toppic here : I question myself about the changes in the help descriptions :
```
| `-d`, `--iteration-data [file]` | Specify a data file to use either json or csv |
```
becomes :
```
  -d, --iteration-data <path>     Specify a data file to use for iterations (either json or csv).
```

Notice the `[]` -> `<>` way to define the data file to use : Does the learning center aims to follow the help page phrasing (that change surely for good reasons), or is it good to reformulate the sentences to help the understanding of the concepts behind options ?

Maintaining documentation up to date is a huge job, but updating litle things each time one can help to follow the stream and don't take too much time :-)

**Comment by SueSmith** [comment]
Hi @Rom1deTroyes! Thanks again and apologies for the delay in replying. Re the help options, I'll pass this back to the engineering team and see if anyone has thoughts on structure etc..

File: src/pages/docs/postman/collection-runs/command-line-integration-with-newman.md [hunk 1]
```diff
@@ -162,6 +163,8 @@ Newman, by default, exits with a status code of 0 if everything runs well, such
 
 Continuous integration tools respond to these exit codes and correspondingly pass or fail a build.
 
+You can use `-x` or `--suppress-exit-code` to override the exit code and have newman exit with a status code of 0, event if tests fails.
```
Comment (SueSmith) [inline src/pages/docs/postman/collection-runs/command-line-integration-with-newman.md]: Could we maybe reword the sentence here to stick more closely to the info in the cli, e.g. "You can use `-x` or `--suppress-exit-code` to override the default exit code for the current run." We could certainly use more examples etc but if we can just get the basic usage covered for now that would be brilliant.

**Comment by SueSmith** [comment]
Pulled this into a fresh branch (#2346) and did that little edit, will be published next time we push to prod - thanks @Rom1deTroyes! 🚀
